### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by cameronbrill**

### CODE REVIEW

File: docs/deploy/configuration.md [hunk 1]
```diff
@@ -71,3 +71,7 @@ This is the API key that Gavel uses to authenticate the sendgrid service.
 #### SECRET_KEY
 
 The secret key is used to hash requests and CSRF tokens. This must be a long, randomly generated alphanumeric string to be effective.
+
+#### VIRTUAL_EVENT
+
+This defines whether your gavel instance is set up for judging a virtual event. Setting this to true produces embedded demo videos and long project description in gavel. Optimized for judging on a computer rather than mobile phone.
```
Comment (mkhatri1) [inline docs/deploy/configuration.md:77]: I think more detail here about the additional fields that virtual mode adds would be nice. This is a list of them:

* Short Tagline
* Submission URL
* Submission Website URL
* Submission Video URL

**Comment by samarthdave** [comment]
Hey @cameronbrill , I wrote some helpful content below. Would you mind adding it into your PR?

## Here is the input that Gavel requests for a project
```
CSV Format: Project Name, Description, Tagline, Video URL, Submission URL, Submission Website
```
### Here's a screenshot of the prompt


**Here's the input I put into the box**

```
Pied Piper,"a powerful compression tool brewed in the mind of Richard Hendricks",
"always blue always blue!","https://www.youtube.com/watch?v=dQw4w9WgXcQ",
"https://devpost.com","https://google.com"
```
I believe the last input "Submission Website" is optional and will be greyed out if it doesn't exist.

## 🎇 Here's the output that is rendered ✨ 


**The "View Full Submission" button goes to devpost.com and the Project Site goes to google.com .

**Comment by samarthdave** [comment]
Okay, I made some changes.

### 🌐 Here is a static link I made to check it out:
https://gavel-docs.surge.sh/deploy/configuration/#virtual-event

@mkhatri1 Can you confirm if it's all correct?

File: docs/deploy/configuration.md [hunk 2]
```diff
@@ -71,3 +71,35 @@ This is the API key that Gavel uses to authenticate the sendgrid service.
...
+```
+CSV Format: Project Name, Description, Tagline, Video URL, Submission URL, Submission Website
+```
+
+| Property           | Description                                       |
+|--------------------|---------------------------------------------------|
+| Project Name       | self-explanatory                                  |
+| Description        | self-explanatory - lies above the buttons         |
+| Tagline            | text to be placed directly below the Project Name |
+| Video URL          | A YouTube link                                    |
```
Comment (cameronbrill) [inline docs/deploy/configuration.md]:
```suggestion
| Video URL          | A video link (Youtube, Vimeo, etc.)  |
```

**Comment by cameronbrill** [comment]
Thank you @samarthdave

**Comment by samarthdave** [comment]
Resolved and updated the [static website](https://gavel-docs.surge.sh/deploy/configuration/#virtual-event).

Also, should I push my `package-lock.json` file? It seems odd that no one else has pushed theirs since it's a pretty important file.

**Comment by cameronbrill** [comment]
they use yarn @samarthdave

File: docs/deploy/configuration.md [hunk 3]
```diff
@@ -71,3 +71,35 @@ This is the API key that Gavel uses to authenticate the sendgrid service.
...
+**Properties for a "Virtual Event"**
+```
+CSV Format: Project Name, Description, Tagline, Video URL, Submission URL, Submission Website
+```
+
+| Property           | Description                                       |
+|--------------------|---------------------------------------------------|
+| Project Name       | self-explanatory                                  |
+| Description        | self-explanatory - lies above the buttons         |
+| Tagline            | text to be placed directly below the Project Name |
```
Comment (mkhatri1) [inline docs/deploy/configuration.md:88]: The Tagline is meant to be "A short, one sentence description for the project"

**Review by mkhatri1 — COMMENTED** [review]
I left a small comment on one of the additions to the configuration page. The rest looks good to me!

File: docs/deploy/configuration.md (hunk 3 above)
Comment (cameronbrill) [inline docs/deploy/configuration.md:88]:
```suggestion
| Tagline            | A short, one sentence description for the project |
```

**Pull request merged**